src/layouts/Dashboard.js

```diff
@@ -77,3 +78,11 @@ export default function Dashboard() {
     </Box>
   );
 }
+
+D3Container.propTypes = {
```

Comment:
Linter is complaining here. `D3Container` should be `Dashboard`

Reply:
Diddidt